Nonprofit-Exchange-Hub/web-app
# Faqs Section

## Dev Summary


[Figma Mocks](https://www.figma.com/file/K7DT5jYcLKRc9epmNrO0N7/UX---Dev-Handoff?node-id=273%3A3675&t=gUF3Yj52JdNpXPNS-0)

Updated the FAQs section of the homepage to reflect the new aesthetic of the Figma mocks. No functionality impacted, primarily just styling. This should provide users with a much more appealing UI. I noticed that others had started to create separate React components for different sections of the homepage, so I followed suit, to help compartmentalize the code. Additionally, I created an Icons folder within Components to store a newly created AccordionIcon (see resource section), that is used to expand the questions within FAQs. I also created the structure for the FAQs image using a placeholder image, to be replaced by the real image, as soon as it is made available.


## Test Plan

No functionality impacted, only styling. Can check that Accordion Icons are working properly.

Repro steps:

1. localhost:3000
2. Scroll to FAQs (above headers) and click on each question
3. Ensure that each question expands, and that the div does not overflow into the Footer (it should expand if necessary)


## Resources

[Stack Overflow - Accordion Icon](https://stackoverflow.com/questions/63691127/how-to-change-icon-when-accordion-summary-is-expanded)
